src/platform_impl/macos/window.rs

```diff
@@ -238,6 +240,18 @@ fn create_window(
             if attrs.position.is_none() {
                 ns_window.center();
             }
+
+            if let Some((r, g, b, a)) = pl_attrs.background_color {
+                let color = NSColor::colorWithSRGBRed_green_blue_alpha_(
+                    nil,
+                    r as f64 / 255f64,
+                    g as f64 / 255f64,
+                    b as f64 / 255f64,
+                    a as f64 / 255f64,
+                );
+                ns_window.setBackgroundColor_(color);
```

Comment:
This is going to be overridden by a `setBackgroundColor_` call in `UnownedWindow::new` further down in this file, when `window.with_transparent(true)` has been called.